_data/tutorials.yaml

```diff
@@ -73,3 +73,12 @@ tumbling-windows:
   status:
     ksql: enabled
     kstreams: enabled
+
+aggregating-minmax:
+  title: "Calculate the number of events over a period of time"
+  slug: "/tutorials/create-stateful-aggregation-minmax"
+  problem: "you have data in a Kafka topic and want to get the minimum and maximum value of a field within the events."
+  introduction: "Suppose you have a topic with events that represent ticket sales of movies. In this tutorial, we'll write a program that calculates the maximum and minimum revenue of movies by year."
```

Comment:
The example for this one is a bit awkward because its a min/max over an aggregation. While the events themselves are just values, it does make you step back and wonder if you're missing something.

Can we switch this up to min/max duration of movies or something? Anything that doesn't imply a pre-aggregated value.

Reply:
I added a bit of narrative to explain the pre-aggregation. I think it works, but lemme know what you think. Will push in a few...